tools/fontbakery-metadata-vs-api.py

```diff
@@ -129,26 +130,26 @@ def getVariantName(item):
                 repo_md5 = hashlib.md5(open(os.path.join(dirpath, repoFileName), 'rb').read()).hexdigest()
 
                 if repo_md5 == google_md5:
-                    log_messages.append([variant, 'OK', '{} in sync'.format(repoFileName)])
+                    log_messages.append([variant, 'OK', '"{}" in sync'.format(repoFileName)])
                 else:
-                    log_messages.append([variant, 'ER', '{}: Checksum mismatch: File in API does not match file in repository'.format(repoFileName)])
+                    log_messages.append([variant, 'ER', '"{}" checksum mismatch, file in API does not match file in git'.format(repoFileName)])
 
             except ValueError:
-                log_messages.append([variant, 'ER', 'Available in API but not in repository'])
+                log_messages.append([variant, 'ER', '"{}" available in git but not in API'.format(font.filename)])
```

Comment:
This should not be inverted this way. This code block is iterating the webFontStyles, so if for a specific style there's an error while looking for its corresponding font in git, then it means it it "available in API but not in git".

As this is the only issue with this code, I'll merge it and then commit a fix to this inversion.